packages/deployment-service/cdk/lib/create-policies.ts

```diff
@@ -1,9 +1,9 @@
-import { Project } from '@aws-cdk/aws-codebuild'
-import { Effect, PolicyStatement } from '@aws-cdk/aws-iam'
-import { DatabaseCluster } from '@aws-cdk/aws-rds'
-import { Bucket } from '@aws-cdk/aws-s3'
-import { ISecret } from '@aws-cdk/aws-secretsmanager'
-import { Queue } from '@aws-cdk/aws-sqs'
+import { Project } from '@reapit/ts-scripts/src/cdk/components/codebuild-project'
```

Comment:
Can you not add the CDK stuff to the ts-scripts manifests so you don't have to provide the full path? No worries if not, just a thought